rockcraft.yaml

```diff
@@ -47,14 +47,14 @@ parts:
       - go/1.19/stable
```

Comment:
Correct me if I'm wrong. It seems that the upstream requires the minimum Golang version to be 1.21. Should we update this line too? https://github.com/glauth/glauth/commit/837da6260792c0ce027643a35b0690ac64ccbed4

Reply:
That's correct, I've just changed it. Thanks for the comment